test/plots/text-overflow.js

```diff
@@ -0,0 +1,138 @@
+import * as Plot from "@observablehq/plot";
+import * as d3 from "d3";
+
+export async function textOverflow() {
+  const names = [
+    "The Best Years of Our Lives",
+    "The Ballad of Gregorio Cortez",
+    "My Big Fat Independent Movie",
+    "Battle for the Planet of the Apes",
+    "Big Things",
+    "Bogus",
+    "Beverly Hills Cop",
+    "Beverly Hills Cop II",
+    "Beverly Hills Cop III",
+    "The Black Hole",
+    "The Big Parade",
+    "Boyz n the Hood",
+    "The Book of Mormon Movie, Volume 1: The Journey",
+    "Return to the Blue Lagoon",
+    "Bright Lights, Big City",
+    "The Blue Bird",
+    "The Blue Butterfly",
+    "Blade Runner",
+    "Bloodsport",
+    "The Blues Brothers",
+    "Blow Out",
+    "De battre mon cœur s'est arrêté",
+    "The Broadway Melody",
+    "Boom Town",
+    "Bill & Ted's Bogus Journey",
+    "The Birth of a Nation",
+    "The Ballad of Cable Hogue",
+    "The Blood of Heroes",
+    "The Blood of My Brother: A Story of Death in Iraq",
+    "Boomerang",
+    "The Bridge on the River Kwai",
+    "Born on the Fourth of July",
+    "Basquiat",
+    "Black Rain",
+    "Bottle Rocket",
+    "👁️‍🗨️👩‍❤️‍💋‍👩👁️‍🗨️👩‍❤️‍💋‍👩👁️‍🗨️👩‍❤️‍💋‍👩👁️‍🗨️👩‍❤️‍💋‍👩👁️‍🗨️👩‍❤️‍💋‍👩👁️‍🗨️👩‍❤️‍💋‍👩", // composed emoji (TODO: slightly buggy)
+    "🧑🏾.👨🏻.👧🏼.👦🏽.🧒🏿.🧑🏾.👨🏻.👧🏼.👦🏽.🧒🏿" // fitz modifiers (TODO: slightly buggy)
+  ];
+  const options = ["clip", "ellipsis", "clip-start", "ellipsis-start", "ellipsis-middle"];
+  return Plot.plot({
+    width: 800,
+    marginLeft: 180,
+    fx: {axis: "top", domain: [...options, "monospace"]},
+    y: {domain: names},
+    marks: [
+      options.map((textOverflow) =>
+        Plot.text(names, {
+          text: names,
+          y: names,
+          fx: () => textOverflow,
+          textOverflow,
+          lineWidth: 8,
+          dx: textOverflow.endsWith("-start") ? 2 : textOverflow.endsWith("-middle") ? 0 : -2,
+          frameAnchor: textOverflow.endsWith("-start") ? "left" : textOverflow.endsWith("-middle") ? "middle" : "right"
+        })
+      ),
+      Plot.text(names, {
+        text: names,
+        y: names,
+        fx: () => "monospace",
+        monospace: true,
+        textOverflow: "ellipsis-end",
+        lineWidth: 13,
```

Comment:
```suggestion
        lineWidth: 13.5,
```

Giving a fractional width here is useful for testing `<` _vs._ `<=` (per previous comment).